### PR TITLE
Remove unused using statements, private fields, and files. 

### PR DESCRIPTION
**Pull request by matthargett**
* Remove unused using statements, private fields, and files.

### CODE REVIEW

File: ReactWindows/ReactNative/Views/ViewManagersPropertyCache.cs [hunk 1]
```diff
@@ -8,11 +8,6 @@ static class ViewManagersPropertyCache
     {
         private static readonly IReadOnlyDictionary<string, IPropertySetter> s_shadowEmpty = new Dictionary<string, IPropertySetter>();
 
-        private static readonly object[] s_viewArgsBuffer = new object[2];
```
Comment (rozele) [inline ReactWindows/ReactNative/Views/ViewManagersPropertyCache.cs]: Suspicious that these are not being used...

Comment (rozele) [inline ReactWindows/ReactNative/Views/ViewManagersPropertyCache.cs]: Not so suspicious after all, must have been left over from when the PropertySetter stuff was refactored.

Comment (rozele) [inline ReactWindows/ReactNative/Views/ViewManagersPropertyCache.cs]: While you're at it, can you move this file back under UIManager folder where it belongs?

**Comment by rozele** [comment]
As long as you're doing this, ReactInlineViewManager.cs -> ReactTextInlineViewManager.cs?

File: ReactWindows/ReactNative/Views/ViewManagersPropertyCache.cs (hunk 1 above)
Comment (matthargett) [inline ReactWindows/ReactNative/Views/ViewManagersPropertyCache.cs]: I'd rather wait to do this when that next set of UIManager and View files are moved to Shared. I did a quick look and I/we can do PropertySetter and the other dependencies once the current set of outstanding PRs are moved.

Comment (kevinvangelder) [inline ReactWindows/ReactNative/Views/ViewManagersPropertyCache.cs]: We can move this back under UIManager in my pull request if you like, as I've already move it to the Shared project under /Views.

**Pull request merged**